src/languages/julia.js

```diff
@@ -84,7 +84,7 @@ function(hljs) {
 
   // placeholder for recursive self-reference
   var DEFAULT = {
-    aliases: ['jldoctest'],
+    aliases: ['julia.console'],
```

Comment:
I'd tend to agree, and also `class: language-julia-console` feels better than `class: language-julia.console` (if I interpret this correctly), but looking around in highlight.js it seems that `.` is generally used for "scoping" related syntaxes together.